tests/authenticate_test.py

```diff
@@ -29,29 +29,42 @@ def testFlaskValidTokenInHeader():
         user = psg.authenticateRequest(request)
         assert user == PASSAGE_USER_ID
 
+def testFlaskInvalidTokenInHeader():
+    psg = Passage(PASSAGE_APP_ID, auth_strategy=Passage.HEADER_AUTH)
+    # flask request context
+    with app.test_request_context("thisisaurl.com",headers={'Authorization':f'Bearer invalid_toke'}):
+        with pytest.raises(PassageError):
+            user = psg.authenticateRequest(request)
```

Comment:
This could also just be
```suggestion
            psg.authenticateRequest(request)
```